MastodonSDK/Sources/MastodonCore/Service/Theme/ThemeService.swift

```diff
@@ -20,7 +20,7 @@ public final class ThemeService {
     public let currentTheme: CurrentValueSubject<Theme, Never>
 
     private init() {
-        let theme = ThemeName(rawValue: UserDefaults.shared.currentThemeNameRawValue)?.theme ?? ThemeName.mastodon.theme
+        let theme = ThemeName.system.theme
```

Comment:
If there's only one theme left... do we still need the `ThemeService`? Or is it just way too much work to get rid of it right away?

Reply:
I thought that it might be handy if we, at some point, maybe decide to bring back different themes, not sure if that's worth it though.